src/libraries/Microsoft.PowerFx.Interpreter/CustomFunction/ReflectionFunction.cs

```diff
@@ -269,6 +269,10 @@ public async Task<FormulaValue> InvokeAsync(IServiceProvider serviceProvider, Fo
                 {
                     arg = FormulaValue.New(string.Empty);
                 }
+                else if (arg is BlankValue && expectedType is RecordType)
+                {
+                    // Add arg as is.
+                }

```

Comment:
Should this be converting blankValue to empty record?

From the above logic, it looks like we are assigning default value in blank case to avoid invocation problem when Execute() method is strictly define.

Ie if you defined input as a record type, then you may as well have taken RecordValue as input, and that would fail if at some point you get BlankValue instead of the RecordValue